.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,38 @@
+name: "Continuous Integration"
+
+on:
+  pull_request:
+  push:
+    branches:
+      - '[0-9]+.[0-9]+.x'
+      - 'refs/pull/*'
+    tags:
+
+jobs:
+  linting:
+    runs-on: "ubuntu-latest"
+    steps:
+      - uses: "actions/checkout@v2"
+      - uses: "actions/setup-node@v2"
+        with:
+          check-latest: true
+      - name: "Install node modules"
+        run: "npm ci"
+      - name: Run ESLint
+        run: "npm run lint"
+  docker-build:
+    runs-on: "ubuntu-latest"
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Build
+        uses: docker/build-push-action@v2
+        with:
+          push: false
```

Comment:
Nice! One note: we have no caching on this, so that will probably be a bit slow, but this component changes rarely (urgh, too rarely, sorry!)

Reply:
Yah I know but would be fine for me. This is just for CI so we can actually use the final image to run integration tests on and afaik its done only once. 
The dependencies are also very limited and thus its lightning fast.